Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject/Definitions/Input.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using static Frends.Salesforce.CreateSObject.Definitions.Enums;
+
+namespace Frends.Salesforce.CreateSObject
+{
+    /// <summary>
+    /// Input-class for CreateSObject-task.
+    /// </summary>
+    public class Input
+    {
+        /// <summary>
+        /// Salesforce Domain.
+        /// /services/data/v52.0/query will be added automatically, so the domain is enough.
+        /// </summary>
+        [DefaultValue(@"https://example.my.salesforce.com")]
+        [DisplayFormat(DataFormatString = "Text")]
+        public string Domain { get; set; }
+
+        /// <summary>
+        /// SObject structure as json.
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Text")]
```

Comment:
Since the user has to input Json string, you should set the default input-type to `JSON`.